src/api/container/opts.rs

```diff
@@ -327,6 +327,11 @@ impl ContainerOptsBuilder {
             .insert("HostConfig.PublishAllPorts", Value::Bool(true));
         self
     }
+    
+    pub fn security_options(mut self, security_options: Vec<String>) -> Self {
```

Comment:
Please consider using the `impl_vec_field` macro as with other vec fields. An example:
```rust
    impl_vec_field!(
        /// Specify any bind mounts, taking the form of `/some/host/path:/some/container/path`
        volumes: V => "HostConfig.Binds"
    );
```
You can see more a little bit further down this file.

A comment would be nice as well, here is something taken directly from Docker api documentation that might work:
```
A list of string values to customize labels for MLS systems, such as SELinux.
```

Reply:
Ok, did that.  Look good?